CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change history for ui-data-import
 
-## 1.5.0 (IN-PROGRESS)
+## [1.5.0](https://github.com/folio-org/ui-data-import/tree/v1.5.0) (2019-09-10)
```

Comment:
I think it also a good practice to add the next version as well `1.6.0 (IN-PROGRESS)`